input/pagecontent/general-requirements.md

```diff
@@ -377,13 +224,16 @@ AU Core PractitionerRole|PractitionerRole.identifier|Medicare Provider Number
 {:.grid}
 
 For example, the profile [AU Core Patient](StructureDefinition-au-core-patient.html) constrains the choices for `Patient.identifier` defined in [AU Base Patient](https://build.fhir.org/ig/hl7au/au-fhir-base/StructureDefinition-au-patient.html) to support Individual Healthcare Identifier (IHI), Medicare Card Number, Department of Veterans' Affairs (DVA) Number. When claiming conformance to the AU Core Patient Profile:
-- AU Core Responders **SHALL** be capable of populating `Patient.identifier` with an IHI, or Medicare Care Number, or DVA Number, or any combination of them
-- AU Core Requesters **SHALL** be capable of meaningfully processing a Patient resource populated with an IHI, Medicare Care Number, and DVA Number.
-- Systems **MAY** support populating and processing other identifiers, but this is not a requirement of AU Core.
+- AU Core Responder **SHALL** correctly populate `Patient.identifier` with an IHI, or Medicare Care Number, or DVA Number, or any combination of them
+- AU Core Requester **SHALL** accept Patient resource resource if `Patient.identifier` is present containing any IHI, Medicare Care Number, or DVA Number identifier type.
+
+Systems **MAY** support populating and processing other identifiers, but this is not a requirement of AU Core.
 
 ##### Must Support - Choice of Profile Elements
 
-A resource may support two elements that are used to indicate a reason, e.g. `Encounter.reasonCode` and `Encounter.reasonReference` in the profile [AU Core Encounter](StructureDefinition-au-core-encounter.html). In such cases, the server **SHALL** be able to populate at least one element, and the client application **SHALL** support processing resources with all supported elements. 
+A resource may support two elements that are used to indicate a reason, e.g. `Encounter.reasonCode` and `Encounter.reasonReference` in the profile [AU Core Encounter](StructureDefinition-au-core-encounter.html). In such cases, the
+- AU Core Responder **SHALL** correctly populate at least one element allowed by the profile, 
```

Comment:
Replace comma at the end of the sentence with ".".

Reply:
fixed.